crates/bevy_pbr/src/prepass/prepass_io.wgsl

```diff
@@ -0,0 +1,101 @@
+#define_import_path bevy_pbr::prepass_io
+
+// Most of these attributes are not used in the default prepass fragment shader, but they are still needed so we can
+// pass them to custom prepass shaders like pbr_prepass.wgsl.
```

Comment:
I wonder how this affects perf, or if the shader compilers can optimize it out 🤔

Reply:
I think this is actually icesentry's comment. Not mine. The existing code in bevy works this way.